Classes/Network/PonyDebugger/FLEXNetworkObserver.m

```diff
@@ -589,9 +589,11 @@ + (void)injectIntoNSURLConnectionCancel {
 
         void (^swizzleBlock)(NSURLConnection *) = ^(NSURLConnection *slf) {
             [FLEXNetworkObserver.sharedObserver connectionWillCancel:slf];
-            ((void(*)(id, SEL))objc_msgSend)(
-                slf, swizzledSelector
-            );
+            if ([slf respondsToSelector:swizzledSelector]) {
+                ((void(*)(id, SEL))objc_msgSend)(
+                    slf, swizzledSelector
+                );
+            }
```

Comment:
I don't think this check is necessary, since `NSURLConnection` will always implement `-cancel`. Do you agree?